third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/transport/bigtable_client_factory.go

```diff
@@ -32,6 +33,10 @@ func (s BigtableClientFactory) NewInstanceAdminClient(project string) (*bigtable
 	opts = append(opts, option.WithTokenSource(s.TokenSource), option.WithUserAgent(s.UserAgent))
 	opts = append(opts, s.gRPCLoggingOptions...)
 
+	if GRPCUnaryClientInterceptor != nil {
```

Comment:
Is this change from the upstream tf repo fetch  or we manually added the logic?

Reply:
Added manually